src/mta-in/entrypoint.pymta.sh

```
@@ -0,0 +1,59 @@
+if [ "${EXEC_CMD_ONLY:-false}" = "true" ]; then
+    exec "$@"
+fi
+
+start_pymta() {
+    python -m pymta.server &
+    PYMTA_PID=$!
+    # Wait until the SMTP port accepts connections (max ~15s). Uses stdlib
+    # socket rather than nc so the runtime image doesn't need netcat just
+    # for this probe.
+    port="${PYMTA_SMTP_PORT:-25}"
+    for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)